samples/java-eventsourced-shopping-cart/src/main/proto/shoppingcart/shoppingcart_api.proto

```diff
@@ -25,55 +25,54 @@ import "google/api/annotations.proto";
 import "google/protobuf/empty.proto";
 
 message AddLineItem {
-  string cart_id = 1 [(akkaserverless.field).entity_key = true];
-  string product_id = 2;
-  string name = 3;
-  int32 quantity = 4;
+    string cart_id = 1 [(akkaserverless.field).entity_key = true];
```

Comment:
Not sure on your editor setup, but the recommended style for protobuf is indent of 2 spaces:

https://developers.google.com/protocol-buffers/docs/style

Reply:
hmm, that's probably badly configured and I probably did the same on the value entity PR.